W-15809945 dw nav proposal duke

dukesphere opened the pull request:
potential options for adding dw mocking lib

dukesphere committed:
W-15809945-dw-nav-duke

dukesphere committed:
W-15809945-dw-nav-duke

dukesphere committed:
Update nav.adoc

suggested mods to ai section

dukesphere committed:
Update nav.adoc

dukesphere commented on modules/ROOT/nav.adoc:
```diff
@@ -65,16 +71,15 @@
 *** xref:int-configure-components-open-from-xml.adoc[]
 ** xref:int-create-secure-configs.adoc[]
 ** xref:int-trigger-flows.adoc[]
```
Removing hidden refs to files that no longer exist:

dukesphere commented on modules/ROOT/nav.adoc:
```diff
@@ -41,10 +41,16 @@
 *** xref:tut-slack-add-condition-to-your-flow.adoc[]
 *** xref:tut-slack-configure-integration.adoc[]
 
+//DW mocking either here after renaming the file to prepend tut- or under the INTEGRATE section (prereq: confirm content needed and up-to-date)
```
Option 1 if we want to retain this topic as a tutorial (vs. adding to the long list of integration topics) 
Note: We're still a bit tbd on what tutorials should be in the TOC and doc sets here vs. another platform.

dukesphere commented on modules/ROOT/nav.adoc:
```diff
@@ -65,16 +71,15 @@
 ** xref:int-import-mule-project.adoc[]
 ** xref:int-deploy-mule-apps.adoc[]
 ** xref:int-versions.adoc[]
+//DW mocking either here after renaming the file to prepend INT- or under the TUTORIALS section (prereq: confirm content needed and up-to-date)
```
Option 2 if we want to retain this topic and don't think it should be considered a tutorial

dukesphere commented on modules/ROOT/nav.adoc:
```diff
@@ -41,10 +41,16 @@
 
+//DW mocking either here after renaming the file to prepend tut- or under the INTEGRATE section (prereq: confirm content needed and up-to-date)
+//** xref:tut-mock-data-using-dw-library.adoc[]
+
```
Suggested consolidation of topic center content so that the top-level node is a high-level JTBD. I believe only overview should be the top-level Overview topic. The rest should be some sort of discrete JTBD or JTBD "bucket",  Reference, or Release Notes. 

That said, I'm just not clear what the rationale is for separating out this topic center overview like this. Maybe there were other issues around this? ¯\_(ツ)_/¯  Maybe the idea was that additional topics would be added below the topic center overview in the future? If so, I'd still argue for a changing the title to something in the JTBD style (Doing X) and clarifying what high-level thing people use topic center for.

dukesphere replied:
If we keep the ai-topic-center.adoc around, perhaps we just change the title to "Enabling API Specs with MuleSoft for Agentforce" I'm told that we need to have MuleSoft for Agentforce in the title.

valkyrie69 replied:
Consolidating the info might be a good option. I'm checking with the PM.

dukesphere replied:
thx, V. I restored the original AI xrefs here so you can do this in another PR.

dukesphere commented on modules/ROOT/nav.adoc:
```diff
@@ -41,10 +41,16 @@
+// Moved the ai topic center content out of the _internal_ DESIGN section of this TOC to its own internal section.
+// After consolidating the overview from xref:ai-topic-center.adoc into ai-enabling-api-project-topic-center.adoc,
+// I'd promote ai-enabling-api-project-topic-center.adoc to an top-level node. 
+* xref:ai-enabling-api-project-topic-center.adoc[]
```
```suggestion
* xref:ai-enabling-api-project-topic-center.adoc[]
```

dukesphere committed:
Update nav.adoc

restoring the AI bit so that V can work on it in a different PR -- expedites that bit

Crispy-Salesforce commented:
Closing this PR since some of the issues were already resolved.
